enterprise_access/apps/api/filters/base.py

```diff
@@ -29,6 +29,6 @@ def get_filterset_class(self, view, queryset=None):
         """
         Returns None if this is a retrieve() operation.
         """
-        if view.action == 'retrieve':
+        if view.action in ('retrieve', 'update', 'partial_update'):
             return None
```

Comment:
outdated docstring

Reply:
fixed in the other PR: https://github.com/openedx/enterprise-access/pull/221/files